app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt

```diff
@@ -78,6 +79,34 @@ class AppVersionActivityTest {
     }
   }
 
+  @Test
+  fun testAppVersionActivity_configurationChange_AppVersionIsDisplayedCorrectly(){
```

Comment:
first character after `_` should be small

`testAppVersionActivity_configurationChange_appVersionIsDisplayedCorrectly`

Reply:
done